Add maintenance schedule check

unka65 opened the pull request:
This adds the maint plugin schedule check.

unka65 committed:
Add maintenance schedule check

netniV commented on README.md:
```diff
@@ -35,6 +35,8 @@ in GitHub.
 
 ## ChangeLog
 
+* feature:  Maintenance schedule check
```
Should have the associated issue number.

netniV commented on webseer_process.php:
```diff
@@ -98,6 +98,17 @@
+	include_once($config['base_path'] . '/plugins/maint/functions.php');
+}
+
+if (function_exists('plugin_maint_check_cacti_host')) {
```
Surely this should check for the correct function `plugin_maint_check_webseer_url` as it could be a version without it?

unka65 replied:
I concur.  I incorrectly thought it was related to the API hook registration.

netniV reviewed (changes requested):
Two minor changes and I think we'll be good to push it in. 👍

unka65 committed:
Maintenance schedule correction

Corrected maintenance schedule condition
Fixed browser warning
Copied page refresh interval from base code, default refresh from config

netniV commented:
Thanks for your hard work, appreciated.
